### PR TITLE
products.js will get and display all products

### PR DESCRIPTION
**Pull request by wvandolah**

### CODE REVIEW

File: src/components/Products.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import axios from "axios";
+import styled from "styled-components";
+
+class Products extends Component {
+  state = {
+    products: []
+  }
+
+  componentDidMount() {
+    axios
```
Comment (NandoTheessen) [inline src/components/Products.js:11]: Great job, William! 
Would love for this to be async / await syntax!

File: src/components/Products.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        this.setState({
+          products: res.data
+        });
+      })
+      .catch(err => console.log(err));
+  };
+
+
+  render() {
+    console.log(this.props)
```
Comment (NandoTheessen) [inline src/components/Products.js:24]: Would you mind removing this console.log?

**Review by NandoTheessen — COMMENTED** [review]
Added a few comments!

**Comment by NandoTheessen** [comment]
References Issue #16

**Comment by NandoTheessen** [comment]
@wvandolah Would you mind tying this into the page, so that we can access the component?

**Comment by wvandolah** [comment]
when you say `tying this into the page` do you just want me to import it the `app.js` and add it to the `render`

**Comment by NandoTheessen** [comment]
Yes sir!

**Pull request closed without merging**